core/services/relay/evm/mercury/v1/report_codec_test.go

```diff
@@ -153,41 +105,24 @@ func Test_ReportCodec_CurrentBlockNumFromReport(t *testing.T) {
 	})
 }
 
-func Test_ReportCodec_FeedIDFromReport(t *testing.T) {
-	r := EVMReportCodec{}
-
-	feedID := utils.NewHash()
-	var validBn int64 = 42
-
-	t.Run("FeedIDFromReport extracts the current block number from a valid report", func(t *testing.T) {
-		report := buildSampleReport(validBn, feedID)
+// func Test_ReportCodec_FeedIDFromReport(t *testing.T) {
+// 	r := ReportCodec{}
 
-		f, err := r.FeedIDFromReport(report)
-		require.NoError(t, err)
+// 	feedID := utils.NewHash()
+// 	var validBn int64 = 42
 
-		assert.Equal(t, feedID[:], f[:])
-	})
-	t.Run("FeedIDFromReport returns error if report is invalid", func(t *testing.T) {
-		report := []byte{1}
+// 	t.Run("FeedIDFromReport extracts the current block number from a valid report", func(t *testing.T) {
+// 		report := buildSampleReport(validBn, feedID)
 
-		_, err := r.FeedIDFromReport(report)
-		assert.EqualError(t, err, "invalid length for report: 1")
-	})
-}
+// 		f, err := r.FeedIDFromReport(report)
+// 		require.NoError(t, err)
 
-func buildSampleReport(bn int64, feedID [32]byte) []byte {
-	timestamp := uint32(42)
-	bp := big.NewInt(242)
-	bid := big.NewInt(243)
-	ask := big.NewInt(244)
-	currentBlockNumber := uint64(bn)
-	currentBlockHash := utils.NewHash()
-	currentBlockTimestamp := uint64(123)
-	validFromBlockNum := uint64(143)
+// 		assert.Equal(t, feedID[:], f[:])
+// 	})
+// 	t.Run("FeedIDFromReport returns error if report is invalid", func(t *testing.T) {
+// 		report := []byte{1}
 
-	b, err := ReportTypes.Pack(feedID, timestamp, bp, bid, ask, currentBlockNumber, currentBlockHash, currentBlockTimestamp, validFromBlockNum)
-	if err != nil {
-		panic(err)
-	}
-	return b
-}
+// 		_, err := r.FeedIDFromReport(report)
+// 		assert.EqualError(t, err, "invalid length for report: 1")
+// 	})
+// }
```

Comment:
Commented code

Reply:
This piece code originally belongs to the orm PR which was not aware of multiple report structure. In order to reduce the dependency between not much related modules, I moved `FeedIDFromReport` from report codec to the orm interface and put the test there. Feel free to revert if that change makes no sense.